From 2d5adcfc824aacc25bbca63849fffd44e3350a6a Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Wed, 16 Sep 2020 11:03:54 +0700
Subject: [PATCH 326/443] arm64: dts: renesas: r8a774e1-hihope-rzg2h-ex: enable
 SATA

In HiHope RZ/G2H, we can choose the output between PCIE1 and SATA by:
- If we choose PCIE1:
	+ Pin 7 of SW1001 on Main Board = ON
	+ SW43 on Sub Board = OFF

- If we choose SATA:
	+ Pin 7 of SW1001 on Main Board = OFF
	+ SW43 on Sub Board = ON

We do not need to set GPIO7_03 to choose PCIE1 or SATA,
so GPIO07_03 is set to input mode.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h-ex.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h-ex.dts b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h-ex.dts
index eef98ef..041ae9b 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h-ex.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h-ex.dts
@@ -22,7 +22,11 @@
 	pcie-en-gpio {
 		gpio-hog;
 		gpios = <3 GPIO_ACTIVE_HIGH>;
-		output-high;
+		input;
 		line-name = "pcie-en-gpio";
 	};
 };
+
+&sata {
+	status = "okay";
+};
-- 
2.7.4

